app/views/common/_discourse_javascript.html.erb

```diff
@@ -46,6 +46,7 @@
     Discourse.LetterAvatarVersion = '<%= LetterAvatar.version %>';
     Discourse.MarkdownItURL = '<%= asset_url('markdown-it-bundle.js') %>';
     Discourse.ServiceWorkerURL = Discourse.Environment != "development" ? '<%= Rails.application.assets_manifest.assets['service-worker.js'] %>' : 'service-worker.js';
+    Discourse.CSPNonce = '<%= nonce %>';
```

Comment:
Expose the nonce to already-executing JS, to enable loading of scripts loaded by another script